src/components/Homepage/MonthlyGoodExplained.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+	<div class="monthly-good-explained">
+		<div class="row">
+			<h1 class="column small-12 text-center">
+				How does Monthly Good work?
+			</h1>
+			<div class="steps-container">
```

Comment:
I would recommend removing this wrapper div since it's not changing the way things are displayed and it would cut down on the number of dom nodes used

Reply:
Removed.